### PR TITLE
Add Repetier-Server component documentation

### PR DESCRIPTION
**Pull request by MTrab**
**Description:**
Documentation for Repetier-Server platform

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

### CODE REVIEW

File: source/_components/repetier.markdown [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+title: Repetier-Server Component
+description: "Instructions how to add Repetier-Server sensors to Home Assistant."
+date: 2019-03-04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_release: 0.88
+ha_iot_class: "Local Polling"
+---
```
Comment (klaasnicolaas) [inline source/_components/repetier.markdown:17]: missing the `ha_category`

**Comment by klaasnicolaas** [comment]
What is the parent PR?

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by MTrab** [comment]
> What is the parent PR?

https://github.com/home-assistant/home-assistant/pull/21658

**Comment by MTrab** [comment]
I do not understand the error thrown - can somebody explain, please?

**Comment by klaasnicolaas** [comment]
Test

**Review by MTrab — COMMENTED** [review]
Comitted

File: source/_components/repetier.markdown [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+layout: page
+title: Repetier-Server Component
+description: "Instructions how to add Repetier-Server sensors to Home Assistant."
+date: 2019-03-04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Hub
```
Comment (klaasnicolaas) [inline source/_components/repetier.markdown]:
```suggestion
ha_category:
  - Hub
  - Sensor
```

File: source/_components/repetier.markdown [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+featured: false
+ha_release: 0.88
+ha_iot_class: "Local Polling"
+---
+
+[Repetier-Server](https://www.repetier-server.com/) is a 3D printer/CNC server, able to control multiple devices on the same server.
+This component handles the main integration to the server.
+
```
Comment (klaasnicolaas) [inline source/_components/repetier.markdown:21]:
```suggestion
There is currently support for the following device types within Home Assistant:

- Sensor
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
After this you can delete the sensor page of the component

And you can add the logo of repetier

File: source/_components/repetier.markdown (hunk 3 above)
Comment (MTrab) [reply source/_components/repetier.markdown:21]: In which path do I put the image for the logo?
Can't seem to find them anywhere.

**Pull request closed without merging**